src/main/scala/org/broadinstitute/hail/expr/Type.scala

```diff
@@ -104,6 +100,12 @@ abstract class Type extends BaseType {
       a => Option(a)
   }
 
+  def printPretty(compact: Boolean = false): String = {
```

Comment:
Can't do this -- overloaded method with different return type doesn't work.  I've renamed it `toPrettyString` for now.